src/components/paint-editor/edit-field-button/edit-field-button.css

```diff
@@ -0,0 +1,31 @@
+@import "../../../css/colors.css";
+@import "../../../css/units.css";
+
+$border-radius: 0.25rem;
+
+.mod-edit-field {
```

Comment:
It's from the trello style guide. If you have a general style for something – in this case, a button – and are applying style that is more specific to a particular use case – here, the edit fields – then the convention is to prefix the class name for that use case with `mod-` to show that it's a modification of the base class.

I realize now that I just kind of did that instinctually since that's how www is set up, but could be off base. Is that not the best way to name this kind of class?